discord_typings/resources/voice.py

```diff
@@ -41,3 +41,13 @@ class VoiceRegionData(TypedDict):
     optimal: bool
     deprecated: bool
     custom: bool
+        
+        
+# https://discord.com/developers/docs/topics/gateway#voice-server-update-voice-server-update-event-fields
+
+
+@final
+class VoiceServerData(TypedDict)
```

Comment:
This class also needs to be added to `__all__`, just add `'VoiceServerData'` to the tuple at the top.

Without doing so, it won't be exposed under the `discord_typings` namespace correctly.